diff --git a/dist/chunks/lib-6EHSwoSb.js b/dist/chunks/lib-6EHSwoSb.js
index 7d9e52f4efdb6bd2def9c9dc235148e339db1bdf..d6ed085edc8d2f95cd5f3c950cf130693c1907aa 100644
--- a/dist/chunks/lib-6EHSwoSb.js
+++ b/dist/chunks/lib-6EHSwoSb.js
@@ -824,7 +824,7 @@ const __filename = import.meta.filename;
 const __dirname = import.meta.dirname;
 const require = __cjs_mod__.createRequire(import.meta.url);
 `;
-const ESMStaticImportRe = /(?<=\s|^|;)import\s*([\s"']*(?<imports>[\p{L}\p{M}\w\t\n\r $*,/{}@.]+)from\s*)?["']\s*(?<specifier>(?<="\s*)[^"]*[^\s"](?=\s*")|(?<='\s*)[^']*[^\s'](?=\s*'))\s*["'][\s;]*/gmu;
+const ESMStaticImportRe = /(?<=^(|\s|;))import\s*([\s"']*(?<imports>[\p{L}\p{M}\w\t\n\r $*,/{}@.]+)from\s*)?["']\s*(?<specifier>(?<="\s*)[^"]*[^\s"](?=\s*")|(?<='\s*)[^']*[^\s'](?=\s*'))\s*["'][\s;]*/gmu;
 function findStaticImports(code) {
     const matches = [];
     for (const match of code.matchAll(ESMStaticImportRe)) {
